### DISCUSSION TITLE
Getting Error for every plugin that I try to open

### DISCUSSION THREAD

**Discussion by vadim9687** [apohl79/audiogridder#1317] [Help]
Server 1.1.1 on Windows 10, Plugin  1.1.1 on Mac Sonoma in Cubase.
every plugin that I try to open thru AudioGridder doesnt open, only gived an error like "failed to add plugin"  "error failed to get result" etc

**Comment by KoiNs257** [comment]
It seens like it can't use in Logic Aboton protools and cubace
